applications/FluidDynamicsApplication/custom_utilities/fluid_auxiliary_utilities.h

```diff
@@ -174,6 +174,20 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) FluidAuxiliaryUtilities
      */
     static ModifiedShapeFunctionsFactoryType GetStandardModifiedShapeFunctionsFactory(const GeometryType& rGeometry);
 
+    /**
+     * @brief This function maps the velocity of the nodes "on the skin" by the use of RBF interpolation
+     * to the variable EMBEDDED_VELOCITY of the nodes which correspond to the cut cells in the volume.
+     * It is designed for use together with the "embedded solver"
+     * @param rVolumeModelPart is the destination domain on which EMBEDDED_VELOCITY will be calculated
+     * @param rSkinModelPart is the skin of the object from which the velocity will be taken
+     * @param SearchRadius is the radius which will be used in searching the neighbours. It needs to be sufficiently large otherwise the method will fail in the calculation of the RBF basis
+     */
+    static void EmbeddedSkinToVolumeMapping(
```

Comment:
```suggestion
    static void EmbeddedVelocityFromSkinToVolumeMapping(
```
IMO the function name is not very well descriptive as it seems to be valid to any set of variables. Indeed it can be easily misunderstood with the `CalculateEmbeddedNodalVariableFromSkinProcess`. I'd suggest to rename it to something like `CalculateEmbeddedVelocityFromSkinVelocityRBF`.

Reply:
i called it MapEmbeddedVelocityFromSkinToVolume